### PR TITLE
Create biopolymers.csl

### PR DESCRIPTION
**Pull request by mfislage**
I started with the ACS style downloaded from zotero style library and tried to make it conform with the style guidelines of biopolymers. 
I...
removed italics and bolds
removed title comments out of references
removed punctuation of abbreviated journal titles
removed the "in" as a prefix for book chapters
removed issue descriptor in jornal articles
changed delimiters in references for journal articles
changed numbering format in reference section
for the book section:
changed delimiters style
removed chapter title

hopefully I did not forget other things I changed.
Formatting for non book and non journal articles is kept in ACS standard ( as recommended)as there is no guideline on how to handle those (since they are generally not used)
Formatting was checked with CSL validator but I am not an expert. So I might have missed things or things could be more elegant

### CODE REVIEW

File: biopolymers.csl [hunk 1]
```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-US">
+  <info>
+    <title>Biopolymers</title>
+    <title-short>Biopolymers</title-short>
```
Comment (adam3smith) [inline biopolymers.csl]: delete this line if it's not different from the regular title

File: biopolymers.csl [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    <id>http://www.zotero.org/styles/Biopolymers</id>
```
Comment (adam3smith) [inline biopolymers.csl]: this is case sensitive -- needs to be `http://www.zotero.org/styles/biopolymers`

File: biopolymers.csl [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%291097-0282/homepage/ForAuthors.html" rel="documentation"/>
+    <author>
+      <name>Marcus Fislage</name>
+      <email>mfislage@gmail.com</email>
+    </author>
+    <contributor>
+      <name></name>
+      <email></email>
+    </contributor>
+    <category citation-format="numeric"/>
```
Comment (adam3smith) [inline biopolymers.csl]: delete lines 16-20 since they're now empty.

File: biopolymers.csl [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+    <eissn>1097-0282</eissn>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%291097-0282/homepage/ForAuthors.html" rel="documentation"/>
+    <author>
+      <name>Marcus Fislage</name>
+      <email>mfislage@gmail.com</email>
+    </author>
+    <contributor>
+      <name></name>
+      <email></email>
+    </contributor>
```
Comment (adam3smith) [inline biopolymers.csl]: delete lines 16-19 since they're now empty.

File: biopolymers.csl [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+        <single>translator</single>
+        <multiple>translators</multiple>
+      </term>
+      <term name="collection-editor" form="short">
+        <single>series ed.</single>
+        <multiple>series eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="editor">
```
Comment (adam3smith) [inline biopolymers.csl]: since you're not using this macro anymore delete it; that's line 42-53

File: biopolymers.csl [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+          <group delimiter=", ">
+            <group delimiter=". ">
+              <text variable="title"/>
+              <text variable="number"/>
+            </group>
+            <date variable="issued" form="text"/>
+          </group>
+        </else-if>
+        <else-if type="chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
+          <group delimiter="; ">
```
Comment (adam3smith) [inline biopolymers.csl]: this got a bit messy. Replace 211-224 with

```
<group delimiter="; ">
  <text macro="book-container"/>
  <text macro="series"/>
  <group delimiter=", ">
     <text macro="publisher"/>
     <text macro="issued"/>
  </group>
  <text macro="volume"/>
  <text macro="pages"/>
</group>
```

which should do the same, just more cleanly and more robustly.

**Comment by adam3smith** [comment]
See comments in line. The automatic failure notice [above](https://github.com/citation-style-language/styles/pull/1767#issuecomment-158693950) has instructions on how to update your pull request. 
Any questions let us know.

**Comment by mfislage** [comment]
I applied all the suggested changes (thanks again) and it is indeed still doing all the formatting as I liked to. 
And it's indeed a lot cleaner

File: biopolymers.csl [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+              <text macro="pages"/>
+            </group>
+        </else-if>
+        <else-if type="webpage">
+          <group delimiter=" ">
+            <text variable="title"/>
+            <text variable="URL"/>
+            <date variable="accessed" prefix="(accessed " suffix=")" delimiter=" ">
+              <date-part name="month" form="short" strip-periods="true"/>
+              <date-part name="day" suffix=", "/>
```
Comment (adam3smith) [inline biopolymers.csl]: sorry, one last bit--I overlooked this: could you remove the space in the suffix here? It's already in the delimiter in 209 above. In Zotero/Mendeley this won't make a difference, but not all products may remove double spaces as reliable.

**Comment by mfislage** [comment]
Done. 
And I am happy about your comments

**Comment by adam3smith** [comment]
great, thanks for putting this up for everyone!

**Comment by adam3smith** [comment]
(It'll appear on zotero.org/styles within an hour; other reference managers may take a little longer).

**Pull request merged**